### DISCUSSION TITLE
It won't disappear after leaving the element, or I can't find the timeout configuration.

### DISCUSSION THREAD

**Discussion by 1014670860** [Akryum/v-tooltip#901] [Q&A]
```
 <Text-Font-Select v-tooltip="$t('tip.fonts')" />
```
This library is really great.
 I am using VUE3 development, but I can't find the timeout setting in the documentation
Mainly because it also causes the parent element to trigger the prompt in the child element, which cannot disappear.

**Comment by Akryum** [comment]
I'm sorry but I don't understand the question. Maybe you can explain a bit more what you are trying to achieve and what your current result is. A runnable example/reproduction would help.

**Reply by 1014670860** [reply]
Is there a timeout configuration item that is automatically hidden after a given time? forgive me for being a rookie.

**Reply by 1014670860** [reply]
Maybe I added the `click` event to `body`, as well as the event modifier `@click.capture`, which caused the problem.
